controllers/infra/drivers/aws/create_instance.go

```diff
@@ -172,7 +172,7 @@ func (d Driver) GetTags(hosts *v1.Hosts, infra *v1.Infra) []types.Tag {
 // get instace data by id
 func getInstancesByID(id *string, client *ec2.Client) ([]types.Reservation, error) {
 	uidKey := fmt.Sprintf("tag:%s", common.InfraInstancesUUID)
-	statusName := common.InstanceState
+	statusName := "instance-state-name"
```

Comment:
"instance-state-name" const should still be const, you can rename it to words like AWSInstanceState